opacclient/gradle.properties

```diff
@@ -11,9 +11,10 @@
 # The setting is particularly useful for tweaking memory settings.
 # Default value: -Xmx10248m -XX:MaxPermSize=256m
 # org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
-org.gradle.jvmargs=-Xmx2048m
+#org.gradle.jvmargs=-Xmx2048m
 
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# org.gradle.parallel=true
+org.gradle.jvmargs=-XX\:MaxHeapSize\=256m -Xmx256m
```

Comment:
It refuses to build and gives this error:

> Gradle sync failed: Unable to start the daemon process.
           This problem might be caused by incorrect configuration of the daemon.
           For example, an unrecognized jvm option is used.
           Please refer to the user guide chapter on the daemon at https://docs.gradle.org/2.10/userguide/gradle_daemon.html
           Please read the following process output to find out more:
           -----------------------
           Error occurred during initialization of VM
           Could not reserve enough space for 2097152KB object heap
It's solved when i add: org.gradle.jvmargs=-XX\:MaxHeapSize\=256m -Xmx256m to gradle.properties. 

The error persists when i add it to local.properties.